vmm/src/vm.rs

```diff
@@ -2658,6 +2658,14 @@ impl Transportable for Vm {
 }
 
 impl Migratable for Vm {
+    fn set_migration_started(&mut self) {
```

Comment:
Perhaps this could just be called `start_migration` and then it would pair with the `complete_migration`

Reply:
Yes, I will modify it.